src/main/java/org/testng/internal/ClassImpl.java

```diff
@@ -184,15 +184,15 @@ private Module newModule(Class<Module> module) {
                   create)
             };
       }
+    }
+    if (m_instances.size() > 0) {
```

Comment:
@juherr - In this situation I believe `size()` method is fine because if I were to be using `isEmpty()` but without the negation (`!`), then I would have flip the conditions. I dont have an issue doing that. But I would like to understand what are we gaining by doing that. The code as it stands IMO is readable and understandable as well.

Reply:
ok